src/index.js

```diff
@@ -6,15 +6,23 @@ exports.register = function (server, options, next) {
   var client = new raven.Client(options.dsn, options.client);
   server.expose('client', client);
   server.on('request-error', function (request, err) {
+    var baseUrl = request.info.uri;
```

Comment:
For parity with the native/express parser please don't use this value unless it's available 100% of the time:

https://github.com/getsentry/raven-node/blob/master/lib/parsers.js#L110